packages/perps-exes/assets/config-testnet.toml

```diff
@@ -1641,5 +1641,5 @@ price = "8"
 price_usd = "8"
 
 [initial-prices.BTC_USDT]
-price = "85000"
-price_usd = "85000"
\ No newline at end of file
+price = "1"
```

Comment:
My comment was only about the price_usd, but I think we can include both changes without breaking anything.